apiserver/README.md

```diff
@@ -16,10 +15,38 @@ managing KubeRay resources are:
 
 KubeRay APIServer maintainer contacts (GitHub handles):
 @Jeffwan @scarlet25151
-```
 
 ## Installation
 
+### Start a local apiserver
+
+You could build and start apiserver from scratch on local environment in one simple command.
```

Comment:
```suggestion
You could build and start apiserver from scratch on your local environment in one simple command. It will deploy all the necessities to a kind cluster.
```

Reply:
Others LGTM.

Reply:
nice wording!